assets/packages/usermanagement/js/usermanagement.js

```diff
@@ -115,8 +116,8 @@ var UserManagement = function () {
                         });
                         return;
                     }
-                    $('#UserManagement--errors').html(LS.LsGlobalNotifier.createAlert(result.errors, 'danger', {showCloseButton: true})
-                    ).removeClass('d-none');
+                    $("#usermanagement-modal-doalog").offset({ top: 10 });
```

Comment:
Typo here? `doalog`

Reply:
Not by be : master : <div id='UserManagement-action-modal' class="modal fade UserManagement--selector--modal" tabindex="-1" role="dialog">

https://github.com/LimeSurvey/LimeSurvey/blob/a3eaefa29d4f4b0790830ae0da78ea385ee426e1/application/views/userManagement/index.php#L83